pkg/master/thirdparty_controller_test.go

```diff
@@ -90,8 +116,8 @@ func TestSyncAPIs(t *testing.T) {
 				Items: resourcesNamed("foo.example.com"),
 			},
 			apis: []string{
-				"/apis/example.com",
-				"/apis/example.com/v1",
+				"/apis/example.com/foos",
```

Comment:
I think I understand what happened-- I think it's not proper to extend paths like "/apis/example.com" to "/apis/example.com/foos". Paths sans version & resource are valid (they should host discovery info) and should exist as long as at least one version exists--that's what you're fixing here as I understand it. But paths with group/resource and no version in between shouldn't exist in the system ever.